tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetCore2.cs

```diff
@@ -27,19 +27,20 @@ public AspNetCore2(ITestOutputHelper outputHelper)
         [InlineData(true, false, HttpStatusCode.OK)]
         [InlineData(false, true, HttpStatusCode.OK)]
         [InlineData(false, false, HttpStatusCode.OK)]
+        [InlineData(true, false, HttpStatusCode.OK, "/Health/?test&[$slice]")]
         [Trait("RunOnWindows", "True")]
         [Trait("Category", "ArmUnsupported")]
-        public async Task TestSecurity(bool enableSecurity, bool enableBlocking, HttpStatusCode expectedStatusCode)
+        public async Task TestSecurity(bool enableSecurity, bool enableBlocking, HttpStatusCode expectedStatusCode, string url = DefaultAttackUrl)
```

Comment:
Maybe I'm missing something, but where is `DefaultAttackUrl` defined?

Reply:
Nevermind, I see that I missed it in `AspNetBase.cs`

Reply:
It is defined in the base class AspNetBase actually, as it's the main url we use for all tests